src/quo2/components/loaders/skeleton/component_spec.cljs

```diff
@@ -0,0 +1,13 @@
+(ns quo2.components.loaders.skeleton.component-spec
+  (:require [quo2.components.loaders.skeleton.view :as skeleton]
+            [quo2.foundations.colors :as colors]
+            [test-helpers.component :as h]))
+
+(h/describe "Skeleton tests"
+  (h/test "Skeleton component is animated when animated? is true"
+    (h/render [skeleton/skeleton-item 0 :messages colors/neutral-10 true])
```

Comment:
Yah, that was the problem I found with the accessibility label, in this case, I will need a different approach or maybe refactor the component. I will come to a solution and reach you to validate it

Reply:
I've come to a solution/workaround for that. Not sure if this is the best so please let me know if u have a better approach. Because at the `ìnternal-view` we are using the `(mod index 4)` I had to make a refactor to generate unique accessibility labels, otherwise, it will repeat and break the test. The problem is that is iterating through each test, which is not my fav.